core/domain/email_manager_test.py

```diff
@@ -187,6 +191,22 @@ def test_email_is_not_sent_if_recipient_has_declined_such_emails(self):
             messages = self.mail_stub.get_sent_messages(to=self.NEW_USER_EMAIL)
             self.assertEqual(len(messages), 0)
 
+    def test_that_email_not_sent_if_can_send_emails_false(self):
```

Comment:
`with_can_send_emails_as_false` or `if_can_send_emails_is_false`. Ditto for all functions.

Reply:
Done